lib/main.js

```diff
@@ -273,23 +273,19 @@ module.exports = class AssetPipeClient {
         } else {
             const writer = new JsWriter(files, options);
 
-            this.transforms.forEach(entry => {
-                writer.transform(entry.transform, entry.options);
-            });
-
             this.plugins.forEach(entry => {
                 writer.plugin(entry.plugin, entry.options);
             });
 
+            this.transforms.forEach(entry => {
+                writer.transform(entry.transform, entry.options);
+            });
+
```

Comment:
were all these order swaps needed in the end?

Reply:
Not entirely sure tbh.